core/controllers/collection_editor.py

```diff
@@ -66,8 +66,8 @@ def get(self, collection_id):
             'SHOW_COLLECTION_NAVIGATION_TAB_STATS': (
                 feconf.SHOW_COLLECTION_NAVIGATION_TAB_STATS),
             'TAG_REGEX': feconf.TAG_REGEX,
+            'title': collection.title,
         })
-
```

Comment:
Undo this change since it is not required. Try removing unnecessary changes by checking them in Files changed tab once you issue a PR.

Reply:
Done. Thanks will keep this in mind for next time.